src/screens/Ticketing/Tickets/Tabs.tsx

```diff
@@ -3,11 +3,18 @@ import Button from '@atb/components/button';
 import {RootStackParamList} from '@atb/navigation';
 import {useRemoteConfig} from '@atb/RemoteConfigContext';
 import {StyleSheet, useTheme} from '@atb/theme';
-import {useTicketState} from '@atb/tickets';
+import {
+  FareContract,
+  FareContractState,
+  filterActiveFareContracts,
+  filterExpiredFareContracts,
+  isPreactivatedTicket,
+  useTicketState,
```

Comment:
Most of these are unused, or?
```suggestion
  filterActiveFareContracts,
  filterExpiredFareContracts,
  useTicketState
```

Reply:
That's right :+1: